### DISCUSSION TITLE
README

### DISCUSSION THREAD

**Discussion by JustIanJ** [codeman1o1/TIJK-Bot#256] [Q&A] [answered]
Why is the "important" section in the readme? Because I have never seen use something like that.

**Comment by karstenzip** [comment]
Good point. I will move it to `CONTRIBUTING.md` later, because it fits better there

**Comment by karstenzip** [comment, marked as answer]
This has been removed in https://github.com/codeman1o1/TIJK-Bot/commit/64c21472e18ced03743414111e969d7439c61ed5
